invenio_rdm_records/records/jsonschemas/records/record-v7.0.0.json

```diff
@@ -0,0 +1,460 @@
+{
```

Comment:
Why do we need this one? it looks like it's the same content as v6, unless I am missing something, see here comparaison: https://www.diffchecker.com/FZPE8bDa/

Reply:
Yes but do we not need to add the same schema for v7 if we are adding v7 mappings? is it different from ILS?